cmd/shipper-state-metrics/collector.go

```diff
@@ -322,6 +332,36 @@ func (ssm ShipperStateMetrics) collectClusters(ch chan<- prometheus.Metric) {
 	}
 }
 
+func (ssm ShipperStateMetrics) collectRolloutBlocks(ch chan<- prometheus.Metric) {
+	nss, err := getNamespaces(ssm.nssLister)
+	if err != nil {
+		glog.Warningf("collect Namespaces: %s", err)
```

Comment:
I'd suggest using glog.Error* family of functions for errors: this is something that goes beyond fyi level of severity.